release_notes/osd_latest_product_updates.adoc

```diff
@@ -62,26 +62,26 @@ details.
 
 [[osd-integrated-metrics]]
 ==== Metrics
-OpenShift Dedicated now includes integrated Hawkular metrics as part of the Base Cluster. This integration allows visibility into memory, CPU, and network usage for each pod.
+{product-title} now includes integrated Hawkular metrics as part of the Base Cluster. This integration allows visibility into memory, CPU, and network usage for each pod.
 
 This metrics integration includes:
 
 .  xref:osd-usability-project-overview[Project Overview] Integration
-.  Metrics tab for specific Pods/Deployments (Applications > Pods > specific pod > Metrics)
+.  Metrics tab for specific pods and deployments (*Applications* -> *Pods* -> specific pod -> *Metrics*)
```

Comment:
@spurtell I put some of the markup for the page titles / navigation back in to match the rest of the topic + elsewhere in the docs.